docs/networking.md

```diff
@@ -290,6 +291,38 @@ Romana uses the cluster's etcd as a backend for storing information about routes
 This does not affect normal etcd operations or require special treatment when upgrading etcd.
 The etcd port (4001) is opened between masters and nodes when using this networking option.
 
+#### Amazon VPC Backend
```

Comment:
Want to make a note that this is in beta?  @liwenwu-amazon on that note, do we want to force the use of a feature flag for this provider?  When is it expected to be production ready?